### DISCUSSION TITLE
2023년 07월 04일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#65] [General]
# 2023년 07월 04일

LOG 베로위키 다시 영업합니다 구경 자주 오세요~ 어제, 그저께, 토요일까지 술을 너무 많이 마셨다 ㅋㅋ.. 술 자제해야 되는데 금요일에도 데모데이 회식이라 술 마시러 간다. 아무래도 자제하자... 백엔드 코드 스타일 컨벤션 회의를 했다. 오랜만에…

https://vero.wiki/2023-07-04/

**Comment by seokjin8678** [comment]
update() 메소드에서 update 쿼리가 왜 안 날아가나요?

**Reply by Cyma-s** [reply]
뭔말이에요

**Comment by kevstevie** [comment]
> 내 안경은 왜 계속 휘어질까

몽둥이찜질

**Reply by Cyma-s** [reply]
브라운한테 말해야겠다~
